lib/rules/lint-simple-unless.js

```diff
@@ -65,18 +122,58 @@ module.exports = class LintSimpleUnless extends Rule {
   }
 
   _withHelper(node) {
-    let loc = node.params[0].loc.start;
-    let actual = `{{unless (${node.params[0].path.original} ...`;
+    let whitelist = this.config.whitelist; // let { whitelist, maxHelpers } = this.config;
+    let maxHelpers = this.config.maxHelpers;
+
+    if (typeof maxHelpers !== 'undefined' || typeof whitelist !== 'undefined') {
```

Comment:
We should ensure that maxHelpers is always initialized up in the config parsing code. We shouldn’t need to compare against undefined....

Reply:
Basically, I think we should remove this conditional completely. As long as `maxHelpers` is initialized to 0 and `whitelist` is initialized to `[]` we can always do the loop / check and still have the same result as before...